RoslynSecurityGuard.Test/Tests/WeakPasswordValidatorAnalyzerTest.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestHelper;
+using Microsoft.CodeAnalysis.Diagnostics;
+using RoslynSecurityGuard.Analyzers;
+using Microsoft.CodeAnalysis;
+using System.Web.Mvc;
+using Microsoft.AspNet.Identity;
+
+namespace RoslynSecurityGuard.Test.Tests
+{
+	[TestClass]
+	public class WeakPasswordValidatorAnalyzerTest : DiagnosticVerifier
+	{
+		protected override IEnumerable<DiagnosticAnalyzer> GetCSharpDiagnosticAnalyzers()
+		{
+			return new[] { new WeakPasswordValidatorAnalyzer() };
+		}
+
+		protected override IEnumerable<MetadataReference> GetAdditionnalReferences()
+		{
+			return new[] { MetadataReference.CreateFromFile(typeof(Controller).Assembly.Location),
+				MetadataReference.CreateFromFile(typeof(System.Linq.Enumerable).Assembly.Location),
+				MetadataReference.CreateFromFile(typeof(PasswordValidator).Assembly.Location) };
+		}
+
+		[TestMethod]
+		public void PasswordValidatorDeclarationTooShort()
```

Comment:
Added a test where the code is secure as requested. I also added a test including the case where the value comes from a variable.